settings.yml

```diff
@@ -138,7 +138,7 @@ big_picture:
     name: Landscape
     short_name: Land...
     title: Interactive Mobility Landscape
-    fullscreen_header: Missing data? Submit a pull request to add! 
+    fullscreen_header: Missing data? Submit a pull request to add! Want us to remove a logo or change an entry? Contact k.shankari@nrel.gov and we will see to it.
```

Comment:
no, don't contact me! Submit a pull request. The whole point of making this a living document is so that people can update it themselves.